app/views/reports/regions/show.html.erb

```diff
@@ -1,117 +1,53 @@
 <%= render "header" %>
 
+<% control_graph_denominator_copy = @with_ltfu ? "denominator_copy" : "denominator_excluding_ltfu_copy" %>
+
 <div class="d-lg-flex flex-lg-wrap">
   <div class="d-lg-flex w-lg-50 pr-lg-2">
-    <div id="bp-controlled" data-period="<%= @period.to_s %>" class="mt-8px mx-0px mb-16px bg-white br-4px bs-small d-lg-flex fd-lg-column justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
+    <div id="bp-controlled"
+         data-period="<%= @period.to_s %>"
+         class="mt-8px mx-0px mb-16px bg-white br-4px bs-small d-lg-flex fd-lg-column
+                justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
       <div class="pt-20px px-20px">
         <div class="d-flex mb-8px">
           <% if current_admin.feature_enabled?(:report_with_exclusions) %>
             <div class="d-flex flex-1">
               <h3 class="mb-0px mr-8px">
                 BP controlled
               </h3>
-              <% if @with_ltfu %>
-                <%= render "definition_tooltip",
-                           definitions: { "Numerator" => t("bp_controlled_copy.numerator"),
-                                          "Denominator" => t("denominator_copy", region_name: @region.name) } %>
-              <% else %>
-                <%= render "definition_tooltip",
-                           definitions: { "Numerator" => t("bp_controlled_copy.numerator"),
-                                          "Denominator" => t("denominator_excluding_ltfu_copy", region_name: @region.name) } %>
-              <% end %>
+              <%= render "definition_tooltip",
+                         definitions: { "Numerator" => t("bp_controlled_copy.numerator"),
+                                        "Denominator" => t(control_graph_denominator_copy, region_name: @region.name) } %>
             </div>
+            <%= render "reports/regions/ltfu_toggle", enabled: @with_ltfu %>
           <% else %>
             <h3 class="mb-0px w-75">
               BP controlled
             </h3>
           <% end %>
```

Comment:
@prabhanshuguptagit maybe I'm missing something, but I don't think the if/else is necessary right. We use the same title, and the ltfu_toggle partial is conditionally rendered already.

Reply:
If the if/else isn't necessary, this comment applies to all other card titles that use it.

Reply:
The if-else toggles the new tooltip and some CSS. The toggle partial isn't conditionally rendered, so we need it for that too.